index.html

```diff
@@ -23,8 +23,8 @@
         </div>
 
         <div class="message">
-            <h3>Spring 2016 General Meetings</h3>
-            <p>For the Spring 2016 semester, the general team meetings will be held on Tuesdays at 6:30 pm in room C-141 in <a href="https://www.wmich.edu/maps/?home=engineering&amp;bldg=214&amp;layer=parking" target="_blank">Floyd Hall</a>.</p>
+            <h3>Fall 2016 General Meetings</h3>
+            <p>For the Fall 2016 semester, the first general team meeting will be held on Wednesday at 6:30 pm in <a href="https://www.wmich.edu/maps/?home=engineering&amp;bldg=214&amp;layer=parking" target="_blank">Floyd Hall</a>. Meeting room is to be announced.</p>
```

Comment:
I would suggest adding the date here, say more than just Wednesday since you're referring to one specific meeting. Make it as clear as possible. @graytwentythree